schemas/src/extensions/stdout/discover.yaml

```diff
@@ -0,0 +1,42 @@
+# yaml-language-server: $schema=https://json-schema.org/draft/2020-12/schema
+$schema: https://json-schema.org/draft/2020-12/schema
+$id:     <HOST>/<PREFIX>/<VERSION>/extension/stdout/discover.yaml
+
+title: Discover extension operation stdout
+description: >-
+  Represents the full path a DSC resource manifest.
+markdownDescription: | # VS Code only
+  ***
+  [_Online Documentation_][00]
+  ***
+
+  Represents the full path a DSC resource manifest. DSC expects every JSON Line
+  emitted to stdout for the **Discover** operation to adhere to this schema. An extension indicates
+  that it supports the **Discover** operation by defining the [discover][01] field in its manifest.
+
+  The output must be a JSON object.
```

Comment:
It may be worthwhile that if nothing is found, then emit nothing to STDOUT and have a 0 exit code.